src/genie/libs/parser/iosxr/tests/ShowImDampeningIntf/cli/equal/golden_output2_expected.py

```diff
@@ -0,0 +1,31 @@
+expected_output = {
+                    'interface': {
+                        'TenGigE 0/1/0/0': {
+                            'currently_suppressed': 'yes',
+                            'dampening_status': 'enabled',
+                            'half_life': 1,
+                            'max_supress_time': 4,
+                            'penalty': 1625,
+                            'protocol': {
+                                'ipv4': {
+                                    'protocol_capsulation': 'ipv4',
+                                    'protocol_penalty': 1615,
+                                    'protocol_suppression': 'no',
```

Comment:
let's take as is. no changing to lower case.
```
`suppression': 'NO'

Reply:
Happy to do that, would you say the rule is leave all values "as is" and not to modify case or wording? I had another case example where a field just said "N" for No so I felt obliged to expand it to "No" so it was clear in terms of meaning. Just curious for anything I do in the future. Thank you.

Reply:
@gwoodwa1 thanks for the comment. actually, we did modification in some parsers in the past. but it created some confusions why it's changed when user is looking for exact output. so we now try to respect the output as is. I think `Y` or `N` is still understandable/guessable. if we had `E`(Enabled) or `D`(Disable), it's a bit hard to guess. In that kind of case, we should consider modification. but usually, will take values as is as much as possible.

Reply:
Cool, thanks for the clarification. That guidance helps alot for the future.

Reply:
confirmed the change.